M2/Macaulay2/m2/examples.m2

```diff
@@ -9,6 +9,7 @@
 
 needs "hypertext.m2"
 needs "run.m2"
+needs "document.m2" -- for DocumentTag
```

Comment:
Ah, this is breaking the strict order of dependencies.

Reply:
Can we just swap `examples.m2` and `document.m2`?

Reply:
That solves part of the problem, but I'm realizing that they've been entangled for a while. It's okay, let's worry about it another day.